chanbackup/backupfile_test.go

```diff
@@ -94,7 +94,7 @@ func TestUpdateAndSwap(t *testing.T) {
 		},
 	}
 	for i, testCase := range testCases {
-		backupFile := NewMultiFile(testCase.fileName)
+		backupFile := NewMultiFile(testCase.fileName, false)
```

Comment:
Maybe it would be the case to test the time stamped channel backup.

Reply:
> Maybe it would be the case to test the time stamped channel backup.

Are you suggesting adding test cases? If I get you right

Reply:
Yes, you got it right.

Reply:
Done, ptal